frappe/database/query.py

```diff
@@ -512,20 +1203,136 @@ def get_query(self, parent_names=None) -> QueryBuilder:
 		)
 
 
+class SqlFunctionParser:
```

Comment:
Lets not parse functions... unnecessary risk for rarely used feature. People should use QB objects and import relevant functions in code.

Reply:
I second this. better to write a separate whitelisted method if functions are really needed.